tests/test_baesian.py

```diff
@@ -0,0 +1,85 @@
+import pytest
+import requests
+import datetime
+from typing import Generator
+
+from utils import MongoDb
+
+baesian_host = "http://localhost:84"
+
+name = "Cicero"
+item_id = 1
+userid_seven = 7
+userid_eight = 8
+
+upsert_data = {
+    "marks": [{"mark": 9, "userid": userid_eight}, {"mark": 9, "userid": userid_seven}],
+    "name": name,
+    "nr_votes": 2,
+    "sum_votes": 18,
+}
+
+
+@pytest.fixture
+def collection() -> Generator[MongoDb, None, None]:
```

Comment:
I would want to refractor this duplicated code after we fi ished writing the tests. The fixture part
I'll think about it. It is ok for now

Reply:
Utils is great!